content/articles/why-kubernetes-is-the-best-technology-for-running-cloud-native-database/index.md

```diff
@@ -0,0 +1,69 @@
+﻿### Introduction
+For a long time, every organization’s dream is to migrate its workload to the cloud. However, the statistics from most IT organizations still show that there is still much work to be done.
+
+Although databases in the cloud have been around for a long time, the challenges with persisting and moving the cloud data have led to slow cloud adoption.
+
+For this reason, the demand for data infrastructure that takes full advantage of cloud computing has been on the rise.
+
```

Comment:
After the above suggested paragraphs, add a new sub-title 'An overview of cloud native databases'. Under this sub-title, provide the definition of a cloud native database and add the other content.